src/components/textarea/textarea.component.js

```diff
@@ -20,196 +20,180 @@ const getFormatNumber = (value, locale) =>
 const marginPropTypes = filterStyledSystemMarginProps(
   styledSystemPropTypes.space
 );
-class Textarea extends React.Component {
-  // Minimum height of the textarea
-  minHeight = 0;
 
-  id = this.props.id || guid();
+const Textarea = ({
+  autoFocus,
+  fieldHelp,
+  label,
+  size,
+  children,
+  characterLimit,
+  enforceCharacterLimit,
+  warnOverLimit,
+  onChange,
+  disabled,
+  labelInline,
+  labelAlign,
+  labelHelp,
+  labelSpacing,
+  inputIcon,
+  id = guid(),
```

Comment:
This will create a new `id` on each render

![texarea-id-change](https://user-images.githubusercontent.com/44157880/140086629-cf7a651c-5c6f-4a85-ad4a-154635788626.gif)

Reply:
@grabkowski :)

Reply:
hahaha